shared/oval/sshd_limit_user_access.xml

```diff
@@ -0,0 +1,28 @@
+<def-group>
```

Comment:
it seems in #1997 only the files for RHEL 7 were moved from `RHEL/7/input/oval/` → `RHEL/7/templates/static/`, not the shared OVALs